### DISCUSSION TITLE
什么时候开始支持 v8 9.4.x

### DISCUSSION THREAD

**Discussion by caichao1103** [Tencent/puerts#1186] [General]
什么时候开始支持 v8 9.4.x , v8 8.4.x 不支持 m1 呀。

**Comment by zombieyang** [comment]
Unity 1.4版本以上用的就是9.4呀

**Reply by caichao1103** [reply]
> Unity 1.4版本以上用的就是9.4呀

那 Unreal Engine 呢？ 谢谢。

**Reply by chexiongsheng** [reply]
UE你直接在这：https://github.com/puerts/backend-v8/releases/tag/V8_9.4.146.24_1115， 下载9.4的替换即可。

**Reply by caichao1103** [reply]
我下载了 https://github.com/puerts/backend-v8/releases/download/V8_9.4.146.24_1115/v8_bin_9.4.146.24.tgz， 这里面的 macOSdylib 文件夹下的文件还是 x86_64 的。哪里可以下到 arm64 的呢。 谢谢。
 
![image](https://user-images.githubusercontent.com/70196128/223603467-8274bfbc-0bba-4bcd-9c50-90660e22cc9e.png)

![image](https://user-images.githubusercontent.com/70196128/223603843-e3e567e9-3d19-4e45-9225-5b7c7d99cfec.png)

**Reply by chexiongsheng** [reply]
Unreal Engine本身出m1版了么？我在m1也是用x86的版本。

**Reply by caichao1103** [reply]
我们是在 macos m1 上源代码编译的 UE 5.1。

**Reply by chexiongsheng** [reply]
backend-v8上macos.sh是x86的静态库，macos_DLL.sh是x86的动态库

有个macos_arm64.sh，是m1静态库，但没m1动态库，你参考这几个加个m1动态库。
